beacon-chain/blockchain/process_block.go

```diff
@@ -84,7 +86,7 @@ func (s *Service) onBlock(ctx context.Context, signed *ethpb.SignedBeaconBlock,
 		return nil, errors.Wrapf(err, "could not save block from slot %d", b.Slot)
 	}
 
-	if err := s.insertBlockToForkChoiceStore(ctx, b, blockRoot, postState); err != nil {
+	if err := s.insertBlockAndAttestationsToForkChoiceStore(ctx, b, blockRoot, postState); err != nil {
```

Comment:
Why did we change this to `insertBlockAndAttestationsToForkChoiceStore`? We dont need to account for attestations here (additional overhead)

Reply:
Just to clarify is there any reason we insert blocks/attestation into fork-choice in initial sync ? The only reason I split it this way was since, I assumed we did need to do it and account for both when syncing as that is the current behavior now.

Reply:
Block is a must, attestation is only optional. But it was so cheap, we ended up doing both anyway to avoid disparity. I don't feel strongly about either, but would prefer clarity (lesser lines and methods). You can decide!